be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/ElectionHandler.scala

```diff
@@ -71,15 +74,84 @@ class ElectionHandler(dbRef: => AskableActorRef) extends MessageHandler {
 
```

Comment:
I tried to test the back end with the front end 1: there is a NullPointerException when we try to fetch the election#setup message from the election channel, because the message is only in the lao channel, so I suggest this change: 
In the handleSetupElection function, to add this at line 54: 
`_ <- dbActor ? DbActor.WriteAndPropagate(electionChannel, message)` 
I tried it and it should work.